docs/production/troubleshooting.md

```diff
@@ -118,9 +118,17 @@ problems and how to resolve them:
   ```
   service nginx restart
   ```
+  Note: Zulip uses nginx, therefore if a new set of certificates are
+
+  being generated and to replace the old certificates, the server 
+
+  will still run the old certificates until the nginx is restarted. 
+
+  **Restarting the Zulip server doesn't restart the nginx server.**
 
 * If your host is being port scanned by unauthorized users, you may see
   messages in `/var/log/zulip/server.log` like
+
```

Comment:
This new line is also not required.

Reply:
Thanks for the heads up! I have removed the extra newlines

Reply:
Kindly remove the line 131 as it is unnecessary. Also, can you please squash all your commits into a single commit? To squash the commits into a single commit, you can refer [this](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit). Also do read [this](https://zulip.readthedocs.io/en/latest/contributing/version-control.html#commit-discipline) for commit discipline.